packages/data-stores/src/reader/queries/use-post-subscriptions-query.ts

```diff
@@ -57,13 +59,25 @@ const usePostSubscriptionsQuery = ( {
 	const outputData = useMemo( () => {
 		// Flatten all the pages into a single array containing all subscriptions
 		const flattenedData = data?.pages?.map( ( page ) => page.comment_subscriptions ).flat();
+
 		// Transform the dates into Date objects
 		const transformedData = flattenedData?.map( ( comment_subscription ) => ( {
 			...comment_subscription,
 			subscription_date: new Date( comment_subscription.subscription_date ),
 		} ) );
-		return transformedData?.filter( filter ).sort( sort );
-	}, [ data, filter, sort ] );
+
+		const searchTermLowerCase = searchTerm.toLowerCase();
+
+		const searchFilter = ( item: PostSubscription ) =>
```

Comment:
I think it could be a value to the user if we considered searching based on the `site_title` and `site_url` - in case they would like to manage post subscriptions related to a certain site.

I was thinking about the `post_excerpt` too, but that might be an overkill.

Reply:
Nice suggestion, I'll add it to the search logic. Thanks @ivan-ottinger!

Reply:
Just updated the PR with the suggested changes, also adding the useSearch to sites tab.